frontend/www/js/omegaup/components/arena/NavbarProblems.vue

```diff
@@ -1,5 +1,12 @@
 <template>
   <div class="problem-list">
+    <div class="breadcurums active" v-if="inAssignment">
+      <span>
+        <a class="name" href="/course/">{{ T.navCourses }}</a> >
+        <a class="name" v-bind:href="urlAssignment">{{ courseName }}</a> >
+        <span>{{ currentAssignment ? currentAssignment.name : '' }}</span>
```

Comment:
```suggestion
        <a class="name" v-bind:href="urlAssignment">{{ courseName }}</a>
        <template v-if="currentAssignment">
          > <span>{{ currentAssignment.name }}</span>
        </template>
```

también, bajo qué circunstancia `currentAssignment` no estaría definido? el tipo de esa variable no es nulable, así que ese `v-if` siempre sería verdadero.

Reply:
Más bien, aquí si debería aceptar `null`, en caso de que sea un concurso. Además, agregué la condición porque me estaba mandando un error que intentaba acceder a la propiedad `name` de un objeto nulo.